### DISCUSSION TITLE
2015-06-09-Talking-about-JavaScript-design-patterns1

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#133] [General]
# 浅谈面向对象的 JavaScript - Tw93

Hi，我是 Tw93，是一个前端工程师，会一点 Swift / Rust / Python，喜欢折腾各种新技术。Technology最近在学习 Ross Harmes/Dustin Diaz 写的《JavaScript 设计模式》，大概将此书看了两遍，很有收获。此书适合于想深入学习JavaScript的同学，用来增加你的内功。能够帮助你增强对JavaScript面向对象的理解，同时学习各种具体的设计模式，懂得应该在什么场合使用这些设计模式，以及如何实现它们，写出更优雅的代码。这篇文章用于总结我的读书笔记 ...

https://tw93.fun/2015-06-09/Talking-about-JavaScript-design-patterns1.html

**Comment by harleyszhang** [comment]
大佬，你这个博客怎么不加个支持目录的功能呢

**Comment by FelixFeli** [comment]
捉个小虫，文章中两处
```
Person.call(this.name);
```
应该是
```
Person.call(this, name);
```
